lib/spells/effects/Moat.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Obstacle.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+#include "Obstacle.h"
+
+VCMI_LIB_NAMESPACE_BEGIN
+
+namespace spells
+{
+namespace effects
+{
+
+class MoatPlacer : public Obstacle
```

Comment:
- file named "Moat.h"
- copyright header says "Obstacle.h"
- class named "MoatPlacer"

Can you unify that?